src/app/series/directives/series-podcast.component.ts

```diff
@@ -117,4 +122,17 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
     }
   }
 
+  get versionTemplateConfirm(): string {
+    if (this.distribution && this.audioVersionOptions) {
+      let url = this.distribution.versionTemplateUrl;
+      let match = this.audioVersionOptions.find(opt => opt[1] === url);
+      let name = match ? match[0] : ''; // options are [[display, value]]
+      return `
+        Are you sure you want to use <b>${name}</b> as the audio for your podcast?
+        Changing this field can have catastrophic consequences for any previously
```

Comment:
Also seems like scare tactics aren't necessary when there are 0 episodes in the series. This can be controlled outside the confirm directive by having an empty string for the confirmation message.